GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -63,6 +65,56 @@ public void RefreshArtificial()
             }
         }
 
+        #region Hotkey commands
+
+        public const string HotkeySettingsName = "BrowseDiff";
+
+        internal enum Commands
+        {
+            DeleteSelectedFiles,
+        }
+
+        protected override bool ExecuteCommand(int cmd)
+        {
+            Commands command = (Commands)cmd;
+
+            switch (command)
+            {
+                case Commands.DeleteSelectedFiles: return DeleteSelectedFiles();
```

Comment:
Just to confirm `DeleteSelectedFiles` only applies to artificial commits?
If so, perhaps it is worth making it more obvious in the names... Maybe something like `DeleteSelectedFilesInArtificialCommit` (the name is kind of ugly, but I can't think of a better one)
It looks like I missed it previously.

Reply:
Delete is always done to the local files.
This action is available to where it is enabled. The menu action is available for artificial (should maybe have been limited to unstaged), the hotkey action can be started for any commit.

The delete for other commits is hard to explain (as it applies to the local file). I have seen this as useful in a few occasions, the check in the delete should maybe be removed. Or limit delete to unstaged only.

I consider the name of the function correct, it describes what it should do and rather remove the artificial check.
However, if this is renamed, should all related functions be renamed too?